### ISSUE TITLE
The label of the LabelWidget show nothing

### ISSUE THREAD

**Issue by tomhaitao** [Kitware/vtk-js#877]

**Comment by jourdain** [comment]
@laurennlam

**Comment by laurennlam** [comment]
Hi @tomhaitao , can you share your code because I already used it and it worked.
@luciemac

**Comment by tomhaitao** [comment]
I'm run LabelWidget example code,but it's nothing show

**Comment by laurennlam** [comment]
Yes, we don't know yet why it doesn't show the label on the example. But it should works if you integrated it into your code.

**Comment by laurennlam** [comment]
Yes, we don't know why it doesn't work on the example. We are working on it.

**Comment by tomhaitao** [comment]
Ok,thank you!

**Comment by nagygylajos** [comment]
Hi @jourdain , @laurennlam , @tomhaitao ,

I tried my best to add labels w/o success.
Could someone be so kind to show me a working example?

Last but not least, IMHO you did a really great library, thank you!
We're really eager to go further on this vtk.js journey.

BR,
^Lajos

**Comment by laurennlam** [comment]
@tomhaitao The LabelWidget has been fixed.
@nagygylajos , you can follow the example to add Labels : https://kitware.github.io/vtk-js/examples/LabelWidget.html
